PackageHandling/Publish-BuildOutputToStorage.ps1

```diff
@@ -58,22 +61,33 @@ try {
             $tempFile = Join-Path (Get-TempDir) "$([Guid]::newguid().ToString()).zip"
             try {
                 Compress-Archive -path (Get-Item (Join-Path $path $_)).FullName -DestinationPath $tempFile
-                Set-AzureStorageBlobContent -File $tempFile -Context $storageContext -Container $projectName -Blob "$appVersion/$_.zip".ToLowerInvariant() -Force | Out-Null
+
+                $version = $appVersion
```

Comment:
```
$versions = @($appVersion)
if ($setLatest) { $versions += @("Latest") }
$versions | % {
   set blob
}
`

Reply:
Yes you are right. setLatest should still upload in a version directory